docsource/intro.rst

```diff
@@ -1,28 +1,103 @@
+In this case, you'll need to perform all intermediate migrations sequentially.
+For exemple, if you want to migrate from version 14.0 to version 16.0, you
+should perform two Upgrades via OpenUpgrade 15.0, then OpenUpgrade 16.0:
+
+.. image:: images/migration-14-16.png
+
+Openupgrade design
+------------------
+
+* From Openupgrade 5.0 to Openupgrade 13.0, the branches
+  in https://github.com/OCA/OpenUpgrade contain copies (or forks in Git terminology)
+  of the Odoo main project, but with extra commits that include the framework,
+  and the analysis and the migration scripts for each module.
+
+* Since Openupgrade 14.0, the branches contain
+
+    * a module ``openupgrade_framework`` which brings together all the patches of odoo
+    * a module named ``openupgrade_scripts`` which contains analysis and migration scripts.
+
+Why migrate to a more recent major version
+------------------------------------------
+
+* `Security holes <https://www.cvedetails.com/vulnerability-list/vendor_id-16543/Odoo.html>`_
+  are regularly found in Odoo.
+  Only the latest versions are patched, so older versions are left with security breeches that can be exploited.
+
+* Odoo regularly works on improving performance, so new versions are
+  `much faster than older ones <https://www.youtube.com/watch?v=uDTKVD-C0eE>`_.
+
+* Odoo is evolving very fast technically.
+  As a result, some previously used technologies are now obsolete.
+  (old python API, Javascript framework replaced by OWL, etc.)
+  If you need to involve developers on your instance,
+  it will be difficult to find someone who is competent and available
+  to work on an instance of an old version.
+
+* Odoo is becoming increasingly popular.
+  As a result, more community modules are available in recent Odoo versions.
+  At the same time, more and more features are available
+  in the native version of odoo.
+
+.. image:: images/oca-modules-evolution.png
+   :alt: : Evolution of the number of OCA modules over the years.
+
+In short, not migrating your Odoo instance to a recent version means increasing
+`your technical debt <https://en.wikipedia.org/wiki/Technical_debt>`_.
 
+Alternative to Openupgrade
+--------------------------
 
-The branches in the first project contain the framework, as well as the
-database analysis and the migration scripts.
+To migrate from one major version to another, there are alternatives to using OpenUpgrade :
 
-The second project contains a library with helper functions. It
-can be used in the migration of any Odoo module.
+1. Odoo S.A.'s service (https://upgrade.odoo.com/).
+   Note that the migration scripts are **not Open Source** and
+   are available are only available if you subscribe to Odoo Enterprise Edition.
+2. Export data (products, customers) from your old instance, install Odoo in a
+   recent version, and import data into the new version.
+   Note that in this case, **you won't recover all the data** in your new instance,
+   but only some of it.
```

Comment:
but only some of it (in particular, you won't have the history and links between data, like the invoices for all customers for example).

Reply:
added.